src/dissectors/ec_o5logon.c

```diff
@@ -46,6 +65,24 @@ void o5logon_init(void);
 
 /************************************************/
 
+#undef memrchr
+#define memrchr my_memrchr
```

Comment:
Just checked FreeBSD. There the `memrchr()` function exists. MacOSX apparently is the only OS lacking `memrchr()` .